go.sum

```diff
@@ -1,3 +1,4 @@
+github.com/Julusian/godocdown v0.0.0-20170816220326-6d19f8ff2df8/go.mod h1:INZr5t32rG59/5xeltqoCJoNY7e5x/3xoY9WSWVWg74=
```

Comment:
Same here, any reason why this file is modified? This is auto-generated so may be a mistaken commit.